### PR TITLE
App Mostly Done with Few Extra Features to be Added

### PR DESCRIPTION
**Pull request by JonathanLadwig**
Any green code or console logs that were left in calendar to work on in the future and will be removed before final commit. Everything otherwise appears to be working with full CRUD, state-management and auth. Here is what it looks like so far:
Screenshot 2023-04-16 at 19 54 12

### CODE REVIEW

File: src/app/Ngrx-store/Ngrx-effects/holiday.effects.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      () =>
+        this.actions$.pipe(
+          ofType(HolidayActions.loadHolidays),
+          switchMap(
+            () =>
+              this.holidayService.getHolidays().pipe(
+                tap((holidays) => console.log(holidays)),
+                map((holidays: IHoliday[]) => HolidayActions.loadHolidaysSuccess({ holidays })),//end of map
+                catchError(error => {
+                  console.log("Hi, yeah we have problems...")
```
Comment (MicNow27) [inline src/app/Ngrx-store/Ngrx-effects/holiday.effects.ts:27]: Remember to remove console.logs before last commit.

File: src/app/components/activity-card/activity-card.component.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  isOpen = false;
+
+  @Input() activity: IActivity | undefined;
+  constructor(private store: Store<ActivityState>, private activityService: ActivityService, private router: Router) { }
+
+  toggle() {
+    this.isOpen = !this.isOpen;
+  }
+
+  cancelDelete() {
```
Comment (MicNow27) [inline src/app/components/activity-card/activity-card.component.ts:44]: Remove unused method

File: src/app/components/calendar/calendar.component.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            (nzPanelChange)="panelChange($event)"></nz-calendar>
+    </div>
+    <br>
+    <h3 class="text-lg">Itinerary for {{selectedDate | date: 'longDate'}}:</h3>
+    <div *ngFor="let activity of allActivities$| async">
+        <app-activity-card [activity]="activity" />
+    </div>
+</div>
+
+<!-- <nz-calendar [(ngModel)]="selectedDate" [(nzMode)]="mode" (nzSelectChange)="selectChange($event)"
```
Comment (MicNow27) [inline src/app/components/calendar/calendar.component.html:13]: Remove commented-out code

File: src/app/components/login/login.component.ts [hunk 4]
```diff
@@ -35,4 +42,11 @@ export class LoginComponent implements OnInit {
   }
 }
 
-
+    // this.auth.fireAuth.currentUser.then(user => {
```
Comment (MicNow27) [inline src/app/components/login/login.component.ts:45]: Remember to remove

File: src/app/services/currency.service.ts [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    const requestURL = `${this.baseConvertCurrenciesURL}from=${fromCurr}&to=${toCurr}&amount=${amount}`;
+    return this.httpClient.get<ICoversionAPIResponse>(requestURL);
+  }
+
+  getSupportedCurrencies() {
+    const requestURL = this.baseSupportedCurrenciesURL;
+    return this.httpClient.get<ICurrencyAPIResponse>(requestURL);
+  }
+
+  // getSupportedCurrencies(): Observable<ICurrencyAPIResponse> {
```
Comment (MicNow27) [inline src/app/services/currency.service.ts:23]: Remove

**Review by MicNow27 — APPROVED** [review]
Nice, approved.

**Comment by JonathanLadwig** [comment]
Dankie Michelle!

**Pull request merged**